schemas/context/segmentdefinition.schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentdefinition",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Definition",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "A Segment Definition schema represents the metadata associated with a segment. For example id, name, description etc",
+  "definitions": {
```

Comment:
Given that the definition block is empty it can be omitted. Of course that begs the question what the purpose of `segmentdefinition` is if it doesn't define additional fields from `segement-meta-base`